src/main/perl/Process.pm

```diff
@@ -103,6 +103,16 @@ regardless of any value for C<NoAction>.
 By default, commands modify the state and thus C<keeps_state> is
 false.
 
+=item C<sensitive>
+
+A boolean specifying whether the command (and args) contain sensitive information
+(like passwords). If C<sensitive> is true, the commandline will not be reported
+(by default the commanlione is reported in with verbose).
```

Comment:
commanlione => command line. Also "by default" do you mean only if the `log` option is not `undef`? Or do you really mean "in all cases unless you set this" ?